scripts/question_answering/run_squad.py

```diff
@@ -311,13 +314,17 @@ def get_network(model_name,
     # Create the network
     Model, cfg, tokenizer, download_params_path, _ = \
         get_backbone(model_name, load_backbone=not backbone_path)
+    cfg.defrost()
+    cfg.MODEL.dtype = dtype
+    cfg.freeze()
```

Comment:
changing it here doesn't seem like a good general approach. would it be better to add the datatype support to `get_backbone`?

Reply:
After a second thought. May be it's better to add dtype to `get_backbone`.

Reply:
ok, np, I made the change back